website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -3749,6 +3749,12 @@ the user ids from a `PCollection` of purchases one would write (using the `Selec
 purchases.apply(Select.fieldNames("userId"));
 {{< /highlight >}}
 
+{{< highlight py >}}
+input_pc = ... # {"userId":...,"bank": ..., "purchase_amount": ...}
+output_pc = input_pc | beam.Map(lambda item: beam.Row(ids=str(item["userId"])))                                            
```

Comment:
```suggestion
input_pc = ... # {"user_id":...,"bank": ..., "purchase_amount": ...}
output_pc = input_pc | beam.Select(user_id=lambda item: str(item["user_id"]))                                            
```

Reply:
Thanks for guiding ,I'll make the appropriate changes